CONTRIBUTING.md

```diff
@@ -321,6 +321,111 @@ development best practices to enhance your Drupal projects.
 that is downloaded when installing a site locally via Lando, or DDev.  See 
 "Visual Studio Code Integration"
 
+## Exporting Configuration
+
+While working on features in local development in Quickstart, there are
+situations where the local build will have configuration changes that are not
+reflected in the saved configuration files of the distribution. This means
+that the changes made will not be reflected in new site builds.
+
+In order for changes to the local site to be reflected in the file tree
+of the feature branch, configuration changes must be exported from the
+development site.
+
+### Exporting Individual Configuration Items
+
+If it is known exactly which configuration entities are changed, they 
+can be exported via the following:
+
+```
+drush az-core-config-export-single name.of.configuration.item
+```
+
+For example, to export `az_cas.settings`, use:
+
+```
+drush az-core-config-export-single az_cas.settings
+```
+
+Note that the `.yml` suffix should not be used when referring to exporting
+these items.
+
+Alternatively, configuration is also available to export through the
+Drupal UI in the local site at the path `/admin/config/development/configuration/full/export`
+or `/admin/config/development/configuration/single/export`
+
+### Exporting Complex Configuration Changes
+
+Some types of changes affect multiple configuration entities. For example, adding new
+fields to a content type typically results in changes to the form display and view display
+of that content type, along with additional field instances and field storage for the fields
+added. It can sometimes be difficult to track down all changes that need to be exported in
+these cases. Quickstart contains a Drush command to automate some of this:
+
+```
+drush az-core-distribution-config
+```
+
+This command examines the site of the local build and compares them to the saved configuration
```

Comment:
In this sentence, maybe "site" could be changed to something like "active configuration items"?